docs/execplans/3-9-2-parse-ninja-status-lines-to-drive-task-progress.md

```
@@ -0,0 +1,406 @@
+6. Update `docs/users-guide.md`, `docs/netsuke-design.md`, and
+   `docs/roadmap.md`.
+7. Run quality gates with `tee` logs:
+
+```sh
+set -o pipefail
+make check-fmt 2>&1 | tee /tmp/3-9-2-check-fmt.log
+make lint 2>&1 | tee /tmp/3-9-2-lint.log
+make test 2>&1 | tee /tmp/3-9-2-test.log
+```
+
+8. Run documentation gates after doc edits:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 8; Style: 1/2/3

(MD029, ol-prefix)